lib/api/apiUtils/object/objectLockHelpers.js

```diff
@@ -158,6 +158,33 @@ function isObjectLocked(bucket, objectMD, headers) {
     return false;
 }
 
+
+/* objectLockRequiresBypass will return true if the retention info change
+ * would require a bypass governance flag to be true.
+ * In order for this to be true the action must be valid as well, so going from
+ * COMPLIANCE to GOVERNANCE would return false unless it expired.
```

Comment:
would going from COMPLIANCE to GOVERNANCE be the only case where action is invalid or is this written as an example?